packages/lit-virtualizer/test/scenarios/testing-setup.test.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import { ignoreWindowErrors, wait } from "../helpers.js";
+import { expect } from "@esm-bundle/chai";
+
+describe('ignoreWindowErrors helper', () => {
```

Comment:
Nit: Some additional tests:
 - A test where the thrown error doesn't match the regex and isn't caught.
 - Maybe a test that checks that `afterEach` is cleaning up.

Reply:
I'm not quite sure how to use Mocha to test Mocha hooks just yet.  But I added a nested case and an explicit error if teardown encounters an unexpected value in window.onerror...